bestie/server/bigquery_metrics.go

```diff
@@ -0,0 +1,288 @@
+package main
```

Comment:
This file can be moved to it's own library package (how about `package metrics` in `lib/metrics`?)

This would allow for:
* shortening of identifiers: `bigQueryMetric` -> `metric.Metric`, `bigQueryTable` -> `metric.Table`, etc (bigQuery is dropped because it's an implementation detail)
* addition of unit tests for these functions

Reply:
Decided to defer this work for now. Not sure of the benefit of making this a separate library, since the code is "BigQuery aware" by definition. This change does not affect the operation of the BES Endpoint and can be considered later on, as time permits. Opened [INFRA-506](https://enfabrica.atlassian.net/browse/INFRA-506) to track as a future work item.